### PR TITLE
sbt: add scalastyle

### LINKED ISSUE #60: Add scalastyle to project
http://www.scalastyle.org/

We should choose witch configuration file is for us. I previously used the twitter one: http://twitter.github.io/effectivescala/#Formatting

### PR DESCRIPTION
**Pull request by erizocosmico**
Closes #60 

Will do a followup PR when #73 is merged to remove all warnings and set that warnings are to be considered errors for CI purposes.

### CODE REVIEW

**Comment by ajnavarro** [comment]
If we found some scalastyle warnings we should stop the travis job, what do you think?

**Comment by erizocosmico** [comment]
Yep, when the scaladoc pr is merged i will fix all warnings and add the setting to consider warnings as errors.

**Comment by erizocosmico** [comment]
Fixed all warnings and moved warnings to errors, ready to review @ajnavarro

File: src/test/scala/tech/sourced/api/util/ColumnFilterSpec.scala [hunk 1]
```diff
@@ -59,17 +59,23 @@ class ColumnFilterSpec extends FlatSpec with Matchers {
     in.matchingCases should be(Map("test" -> Seq("a", "b", "c")))
 
     in.eval(Map("test" -> "a")) should be(Some(true))
-    in.eval(Map("test" -> "a",  "foo" -> "a")) should be(Some(true))
+    in.eval(Map("test" -> "a", "foo" -> "a")) should be(Some(true))
     in.eval(Map("test" -> "d")) should be(Some(false))
     in.eval(Map("foo" -> "b")) should be(None)
-  }
+  }º
```
Comment (ajnavarro) [inline src/test/scala/tech/sourced/api/util/ColumnFilterSpec.scala]: what does it means?

Comment (erizocosmico) [reply src/test/scala/tech/sourced/api/util/ColumnFilterSpec.scala]: a typo :stuck_out_tongue:

**Pull request merged**